static/app/utils/events.tsx

```diff
@@ -178,7 +178,7 @@ export function getTitle(
       };
     case EventOrGroupType.DEFAULT:
       return {
-        title: customTitle ?? metadata.title ?? '',
+        title: customTitle ?? title,
```

Comment:
As a reference, this is what `customTitle` is set:
https://github.com/getsentry/sentry/blob/c9b49d8c538c58a2a010e714debcb008c4ff1043/static/app/utils/events.tsx#L129

`''` also yields the same problem, thus, we fallback to `title` instead of `''`.